<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ShadowingDialog</class>
 <widget class="QDialog" name="ShadowingDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>255</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>600</width>
    <height>255</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>600</width>
    <height>255</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>550</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string comment="Configure Shadowing Timespan" >Shadowing Period</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="QWidget" name="verticalLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>180</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Number of Slides</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading</set>
      </property>
      <widget class="QSpinBox" name="slidesCount" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum" >
        <number>255</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>272</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_5" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>130</width>
          <height>125</height>
         </size>
        </property>
        <property name="title" >
         <string>Start</string>
        </property>
        <widget class="QLabel" name="label_6" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>90</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text" >
          <string>Time</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>30</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text" >
          <string>Date</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="startDate" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>50</y>
           <width>111</width>
           <height>19</height>
          </rect>
         </property>
         <property name="displayFormat" >
          <string>dd MMM yyyy</string>
         </property>
        </widget>
        <widget class="QTimeEdit" name="startTime" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>110</y>
           <width>111</width>
           <height>19</height>
          </rect>
         </property>
         <property name="time" >
          <time>
           <hour>12</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="displayFormat" >
          <string>hh:mm:ss</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="minimumSize" >
         <size>
          <width>130</width>
          <height>125</height>
         </size>
        </property>
        <property name="title" >
         <string>End</string>
        </property>
        <widget class="QLabel" name="label_4" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>30</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text" >
          <string>Date</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>90</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text" >
          <string>Time</string>
         </property>
        </widget>
        <widget class="QTimeEdit" name="endTime" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>110</y>
           <width>111</width>
           <height>19</height>
          </rect>
         </property>
         <property name="time" >
          <time>
           <hour>13</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="displayFormat" >
          <string>hh:mm:ss</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="endDate" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>50</y>
           <width>111</width>
           <height>19</height>
          </rect>
         </property>
         <property name="displayFormat" >
          <string>dd MMM yyyy</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayout_2" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>10</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="minimumSize" >
       <size>
        <width>130</width>
        <height>125</height>
       </size>
      </property>
      <property name="autoFillBackground" >
       <bool>false</bool>
      </property>
      <property name="title" >
       <string>Geographic Position</string>
      </property>
      <widget class="QLabel" name="label" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string>Longitude</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>90</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text" >
        <string>Latitude</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="longitudeDeg" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>50</y>
         <width>58</width>
         <height>19</height>
        </rect>
       </property>
       <property name="suffix" >
        <string>°</string>
       </property>
       <property name="maximum" >
        <number>180</number>
       </property>
       <property name="minimum" >
        <number>-180</number>
       </property>
       <property name="value" >
        <number>11</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="longitudeMin" >
       <property name="geometry" >
        <rect>
         <x>67</x>
         <y>50</y>
         <width>44</width>
         <height>19</height>
        </rect>
       </property>
       <property name="suffix" >
        <string>'</string>
       </property>
       <property name="prefix" >
        <string/>
       </property>
       <property name="maximum" >
        <number>59</number>
       </property>
       <property name="value" >
        <number>34</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="longitudeSec" >
       <property name="geometry" >
        <rect>
         <x>110</x>
         <y>50</y>
         <width>44</width>
         <height>19</height>
        </rect>
       </property>
       <property name="suffix" >
        <string>''</string>
       </property>
       <property name="maximum" >
        <number>59</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="latitudeDeg" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>110</y>
         <width>58</width>
         <height>19</height>
        </rect>
       </property>
       <property name="readOnly" >
        <bool>false</bool>
       </property>
       <property name="suffix" >
        <string>°</string>
       </property>
       <property name="maximum" >
        <number>90</number>
       </property>
       <property name="minimum" >
        <number>-90</number>
       </property>
       <property name="value" >
        <number>48</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="latitudeMin" >
       <property name="geometry" >
        <rect>
         <x>67</x>
         <y>110</y>
         <width>44</width>
         <height>19</height>
        </rect>
       </property>
       <property name="suffix" >
        <string>'</string>
       </property>
       <property name="prefix" >
        <string/>
       </property>
       <property name="maximum" >
        <number>59</number>
       </property>
       <property name="value" >
        <number>8</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="latitudeSec" >
       <property name="geometry" >
        <rect>
         <x>110</x>
         <y>110</y>
         <width>44</width>
         <height>19</height>
        </rect>
       </property>
       <property name="suffix" >
        <string>''</string>
       </property>
       <property name="maximum" >
        <number>59</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayout_3" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>180</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_4" >
      <property name="title" >
       <string>Time Zone</string>
      </property>
      <widget class="QSpinBox" name="timeZone" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>19</height>
        </rect>
       </property>
       <property name="maximum" >
        <number>12</number>
       </property>
       <property name="minimum" >
        <number>-12</number>
       </property>
       <property name="value" >
        <number>1</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>466</x>
     <y>10</y>
     <width>121</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton" >
        <property name="text" >
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>startDate</tabstop>
  <tabstop>startTime</tabstop>
  <tabstop>endDate</tabstop>
  <tabstop>endTime</tabstop>
  <tabstop>longitudeDeg</tabstop>
  <tabstop>longitudeMin</tabstop>
  <tabstop>longitudeSec</tabstop>
  <tabstop>latitudeDeg</tabstop>
  <tabstop>latitudeMin</tabstop>
  <tabstop>latitudeSec</tabstop>
  <tabstop>slidesCount</tabstop>
  <tabstop>timeZone</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
